src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -1311,7 +1362,7 @@ public void SetBeforeAccess(TokenCacheCallback beforeAccess)
         public void SetAfterAccess(TokenCacheCallback afterAccess)
         {
             GuardOnMobilePlatforms();
-            AfterAccess = afterAccess;
```

Comment:
These should remain since the implementation of AfterAccess {set;} just does _userConfiguredAfterAccess = afterAccess anyway.  Then, if we add any logic (locking, logging, etc) to the AfterAccess setter it will take effect for both cases.